books/spiders/books_info.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any
+import re
+
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class BooksInfoSpider(scrapy.Spider):
+    name = "books_info"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self, **kwargs: Any):
+        super().__init__(**kwargs)
+        options = webdriver.ChromeOptions()
+        options.add_argument("--headless")
+        self.driver = webdriver.Chrome()
+
+    def close(self, reason) -> None:
+        self.driver.close()
```

Comment:
Consider using `self.driver.quit()` instead of `self.driver.close()` to ensure that all browser windows are closed and the WebDriver session is properly terminated.